dts/bindings/sensor/memsic,mmc56x3.yaml

```diff
@@ -0,0 +1,39 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (c) 2023 Linumiz
+
+description: MMC56X3 3-axis magnetic and temperature sensor
+
+compatible: "memsic,mmc56x3"
+
+include: [sensor-device.yaml, i2c-device.yaml]
+
+properties:
+
+  magn-odr:
+    type: int
+    default: 128
```

Comment:
Need to explain in the description why the default value was selected
https://docs.zephyrproject.org/latest/build/dts/bindings-upstream.html#dt-bindings-default-rules

Reply:
Removed default because I couldn't give a reason to pick any number (none is recommended)